spec/models/user_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'associations' do
+    it { should have_many(:foods) }
+    it { should have_many(:recipes) }
```

Comment:
- The same issue is applicable here and in line 16. Please, verify to fix it.

![image](https://github.com/0sugo/recipe-app/assets/59397159/ea72062f-5c19-40c8-9c0a-d8c14c775c4b)

Reply:
Thanks @zieeco .Resolved